README.md

```diff
@@ -4,10 +4,35 @@ Website for Digital Twin as a Service (DTaaS) software. This software shall be u
 
 This is a mono repo containing code for both the client (web browser) and server code.
 
+## Before Installing the Environment
+
+Give Linux user sudo privileges, if your user already has sudo privileges you can skip this step. <br />
+
+1. On start up hold shift. <br />
+2. Select the second option, "Advanced options for Ubuntu". <br />
+3. Select the second option, "Ubuntu, with Linux 5.15.0-60-generic (recovery mode)". <br />
+4. On the "Recovery Menu" select "root" and enter your password. <br />
+5. Use the following:
+```bash
+usermod -aG sudo <username>
+exit
+```
+6. Select "resume". <br />
+
+Optional: Once you're back on the desktop check if your user has sudo privileges:
+```bash
+groups <username>
+```
+7. Log in to git using cmd line:
+```bash
+git config --global user.name "your_username"
+git config --global user.email "your_email_address@example.com"
+```
+
```

Comment:
This is the top-level README. Are the above instructions relevant to the whole project?

Reply:
We agree, this is not that relevant. We will remove it :)